fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -4471,19 +4372,6 @@ public void testContainerLastModified() throws Exception {
                 headerFormat.parse(response.getFirstHeader("Last-Modified").getValue())).toEpochMilli();
             assertTrue(lastmod3 > lastmod2);
         }
-
-        sleep(1000); // wait a second to make sure last-modified value will be different
-
-        // create child in the container
-        assertEquals(CREATED.getStatusCode(), getStatus(new HttpPut(objURI + "/members/member1")));
-
-        // last-modified should be updated
```

Comment:
We don't (and won't) update the Last-Modified date of the object when a new membership property appears. This will however update the ETag.